docs/kit/logistics/kit-collation/README.md

```diff
@@ -5,6 +5,10 @@ original:
 ---
 # Kit Collation
 
+!!! note
```

Comment:
Nit: I don't think this statement is necessary. There's more to kit collation than just testing.

Given there's a "Test" section below, I'd be open to replacing that entirely with a link out to the new page.

Reply:
The new kit testing page encompasses so much more than what test covers here. It now encorporates check-in and checkout and should be done before the visual check.